### DISCUSSION TITLE
Newbie question: switch device from Read to Controllable in Cloud Project

### DISCUSSION THREAD

**Discussion by dgrillo-github** [jasonacox/tinytuya#417] [General]
Hi tinytuya users, I'm using tinytuya to locally control some Feit brand Color Lights I recent purchased.   I have four lights, and three are working completely correctly;  I'm able to control them using the local api.   The fourth, however, isn't working correctly, and I think the problem is it is showing up as Device Permission Read vs. Device Permission Controllable in the IoT website.   I would have thought this would be easy to change in the UI but I can't find it...

**Comment by jasonacox** [comment]
It seems odd that this is the only device not working. Are you able to read from the device locally (e.g. `status()`)?

**Comment by uzlonewolf** [comment]
Whelp, while poking around trying to find the permissions selector I managed to make all my devices read-only :rofl: It seems they removed the device permissions selector when they re-worked the API a few months ago.  Even though my devices are now showing as read-only they still seem to work fine.  What error are you seeing @dgrillo-github ?
